src/EventListener/SavePropertyListener.php

```diff
@@ -50,6 +55,10 @@ public function onCategoryPostSave(GenericEvent $event): void
             return;
         }
 
+        if ($this->validator->validate($properties) !== []) {
+            throw new RuntimeException('Invalid properties format');
```

Comment:
`RuntimeException` is very generic, maybe consider using or creating a more specific exception type.

Reply:
I've added new Exception classes according to the coding style.